content/en/blog/2020/addon-rework/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Reworking our Addon Integrations
+description: A new way to manage installation of telemetry addons.
+publishdate: 2020-06-10
+attribution: John Howard (Google)
+keywords: [telemetry,addons,integrations,grafana,prometheus]
+target_release: 1.6
+test: n/a
+---
+
+In Istio 1.6, we are introducing a new method for integration with telemetry addons (Grafana, Prometheus, Zipkin, Jaeger, Kiali, etc).
+
+In previous releases, these addons were bundled as part of the Istio installation. At the time, this was incredibly useful as it allowed users to quickly get started without dealing with complicated configurations to install and integrate with these applications. However, it came with a number of issues. Our installations were not as up to date or feature rich as upstream installation methods, meaning users were left missing out on some of the great features provided by these applications such as persistent storage, features like `Alertmanager` for Prometheus, and advanced security settings. Additionally, integration with existing deployments that were using these features was more challenging than it should be.
+
+## Changes
+
+In order to address these gaps, we have made a number of changes:
+* Added a new [Integrations](/docs/ops/integrations/) documentation section to explain what applications Istio can integrate with, how to use them, and best practices.
```

Comment:
All of these bullets are not displaying correctly (check the preview). Need blank lines before and proper 4 space indent of second level items.

Reply:
thanks. fixed